contents/blog/eu-cloud-hosting.md

```diff
@@ -0,0 +1,89 @@
+---
+date: 2022-10-17
+title: How we built EU Cloud hosting, from idea to launch
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog", "Startups"]
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+---
+
+We recently launched EU Cloud, providing all the benefits of cloud-hosted PostHog with data staying in the EU. We thought it would be interesting to go over why and how we built EU Cloud hosting, from idea to launch.
```

Comment:
We should also remove the passive voice and clear up the tenses here. 

```suggestion
We recently launched PostHog Cloud EU, which provides all the benefits of a cloud-hosted PostHog, but keeps data in the EU. We thought it would be interesting to go over why and how we built PostHog Cloud EU, from idea to launch.
```